hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/tools/DebugAdmin.java

```diff
@@ -432,8 +432,16 @@ private class VerifyECCommand extends DebugCommand {
 
     VerifyECCommand() {
       super("verifyEC",
-          "verifyEC -file <file>",
-          "  Verify HDFS erasure coding on all block groups of the file.");
+          "verifyEC -file <file> [-blockId <blk_Id>] [-ignoreFailures]",
```

Comment:
Can change `-ignoreFailures` to `-verifyAllFailures`?

Reply:
If I understand correctly, the command with `-ignoreFailures` always returns 0. If we change it to `-verifyAllFailures`, I think it should return 1 if there are one or more failures.

Reply:
Thanks @tasanuma @ZanderXu help me to review it.

Add option -ignoreFailures  to control whether to ignore failures block group . If set -ignoreFailures, will ignores failures block group during verify and continues verify all block groups of the file.

Reply:
Whether -ignoreFailures is set or not, it should return 1 if there are one or more block group failures

Reply:
change -ignoreFailures to -verifyAllFailures might express this better.

Reply:
if the option is specified, it skips this if clause, and it should always return 0, shouldn't it?
```java
            if (!verifyAllFailures) {
              return 1;
            }
```

Reply:
May `-ignoreFailures` is better? And we can keep the previous change `return 1`.

Reply:
Thanks @tasanuma @tomscut  help me to review it. 
yeah , we need to keep the previous return value logic, if verify failure return 1, else return 0.

Reply:
If the option `-verifyAllFailure`s is not specified, if the all block groups of file is healthy, actually it will verify all blocks.
so for `verifyAll`, maybe not very easy to understand.

@ZanderXu @tomscut @tasanuma  how about change to  -skipFailureBlocks ?

Reply:
@haiyang1987 `-skipFailureBlocks`  seems good to me. If we use the words "skip" or "ignore", I don't mind the return value. (I mean, either "always return 0" or "return 0 if there are not any failures, else return 1" would be fine.)

Reply:
> If the option `-verifyAllFailures` is not specified, if the all block groups of file is healthy, actually it will verify all blocks. so for `verifyAll`, maybe not very easy to understand.
> 
> @ZanderXu @tomscut @tasanuma how about change to -skipFailureBlocks ?

I think `-skipFailureBlocks` is ok.